server/routes/admin.js

```diff
@@ -1,16 +1,42 @@
 import { Router } from 'express'
+import * as yup from 'yup'
 
 import ensureAdmin from '../utils/passport/ensure-admin'
 import AdminUsersController from '../controllers/adminController'
 import { v1Routes } from '../utils/routes'
 import StudiesController from '../controllers/studiesController'
+import validateRequest, { baseSchema } from '../middleware/validateRequest'
 
 const router = Router()
+const uid = yup.string().required()
```

Comment:
What do you think about keeping our schemas in a separate directory where they each get their own file? I think we could potentially reuse schemas that way.

Reply:
I battled with this. I'm still unconvinced about having a individual files for each schema. I kept them in the same files because I feel they don't take up much space. are you opposed to one file that exports the schemas..?

Reply:
Kinda, yeah, that file would get big. I was thinking one file per schema would let us share and compose them, so we could use for example the `userSchema` in the Admin view and the Profile view. But if schemas are really distinct I don't mind this approach, and definitely prefer the current code to one big schemas file.